### ISSUE TITLE
CMake errors while building on Windows 10

### ISSUE THREAD

**Issue by wyatt-sfu** [rnpgp/rnp#1927]
## Description
While following the Windows build instructions in [https://github.com/rnpgp/rnp/blob/master/docs/installation.adoc](https://github.com/rnpgp/rnp/blob/master/docs/installation.adoc) I am getting CMake failures with a couple of different error messages. 

## Steps to Reproduce
1. Download and unzip rnp-0.16.2
2. Install vcpkg with `git clone https://github.com/Microsoft/vcpkg.git`
3. Install packages with `vcpkg install bzip2 zlib botan json-c getopt dirent python3[core]`
4. Run `cmake -B . -G "Visual Studio 16 2019" -A x64 -DCMAKE_TOOLCHAIN_FILE=%VCPKG_ROOT%\scripts\buildsystems\vcpkg.cmake -DCMAKE_BUILD_TYPE=Release ../rnp`

## Versions
- vcpkg git hash: 7ad236f60f5f7197e93c4d7f0807622f4899076d
- cmake: 3.16.2
- rnp: 0.16.2

## Results
CMake output: [rnp_cmake_error.txt](https://github.com/rnpgp/rnp/files/9861546/rnp_cmake_error.txt)
I have attached the CMake log as well: [CMakeOutput.log](https://github.com/rnpgp/rnp/files/9861407/CMakeOutput.log)
vcpkg list: [vcpkg_list.txt](https://github.com/rnpgp/rnp/files/9861578/vcpkg_list.txt)

A snippet from the output is:
```
CMake Error at C:/Program Files/CMake/share/cmake-3.16/Modules/FindPackageHandleStandardArgs.cmake:146 (message):
  Could NOT find JSON-C (missing: JSON-C_LIBRARY JSON-C_INCLUDE_DIR
  JSON-C_VERSION) (Required is at least version "0.11")
Call Stack (most recent call first):
  C:/Program Files/CMake/share/cmake-3.16/Modules/FindPackageHandleStandardArgs.cmake:393 (_FPHSA_FAILURE_MESSAGE)
  cmake/Modules/FindJSON-C.cmake:92 (find_package_handle_standard_args)
  C:/dev/vcpkg_rnp/scripts/buildsystems/vcpkg.cmake:834 (_find_package)
  src/lib/CMakeLists.txt:33 (find_package)
```

**Comment by wyatt-sfu** [comment]
@ni4 I think I have figured out the issue. I did not need a VCPKG_DIR environment variable because the build tutorial uses VCPKG_ROOT.

However, the trick was to specify the x64-windows triplet when installing the VCPKG packages. So rather than
`vcpkg install bzip2 zlib botan json-c getopt dirent python3[core]`, 
I instead ran,
`vcpkg install bzip2:x64-windows zlib:x64-windows botan:x64-windows json-c:x64-windows getopt:x64-windows dirent:x64-windows python3[core]:x64-windows`
After this change, then running cmake and building rnp seems to have worked fine. I can run rnp.exe and rnpkeys.exe.
